_dashboards/geojson-regionmaps.md

```diff
@@ -31,11 +31,15 @@ To create your own custom vector map, upload a JSON file that contains GEO data
 1. Choose a source. For example, **[Flights] Flight Log**.
 1. In the right panel, select **Import Vector Map**. 
 1. In **Upload map**, select or drag and drop your JSON file.
-Enter **Map name prefix** (for example, `us-county`). Your map will have the prefix that you defined followed by the `-map` suffix (for example, `us-county-map`). 
-Select the **Import file** button. 
-Once the upload is successful, you will see a pop-up prompting you to refresh the map. Select the **Refresh** button.
+Enter **Map name prefix** (for example, `usa-county`). Your map will have the prefix that you defined followed by the `-map` suffix (for example, `usa-county-map`). <img src="{{site.url}}{{site.baseurl}}/images/import-geojson-file.png" alt="import a Geo .json file" width="340"/>
+1. Select the **Import file** button.
+Once the upload is successful, you will see a pop-up prompting you to refresh the map. Select the **Refresh** button. <img src="{{site.url}}{{site.baseurl}}/images/upload-success.png" alt="message upon a successful file upload" width="280"/>
+1. You can now view your region map in the Dashboards. For example, the following image shows the Los Angeles county region: 
```

Comment:
To view your region map in the Dashboards, follow steps 5 and 6 in the Visualize your region map section.